### PR TITLE
Create Recommendations

### PR DESCRIPTION
**Pull request by gdalvi**
- this is a bullet
- [ ] Task list

### CODE REVIEW

**Comment by bhediathewolf** [comment]
Hi gdalvi, I am Sannidhya from New Delhi, India. Sup?

File: Recommendations [hunk 1]
```diff
@@ -0,0 +1 @@
+So far so good.
```
Comment (gdalvi) [inline Recommendations:1]: Look it up

**Comment by gdalvi** [comment]
General commment

**Review by gdalvi — COMMENTED** [review]
Done

**Review by bhediathewolf — COMMENTED** [review]
Okay, nice.

**Review by gdalvi — COMMENTED** [review]
Approve the changes please

**Pull request merged**